docs/doc.main

```diff
@@ -146,12 +146,20 @@
 
 
   "General" => [
-     "General/styleguide.md",
      "General/other.md",
      "General/faq.md",
-		 "General/SubObjectIterator.md"
   ],
 
   "References" => "references.md",
   "Index" => "manualindex.md",
+  "Developers Corner" => [
+		"DevelopersCorner/intro.md",
+		"General" => [
+			"DevelopersCorner/styleguide.md",
+			"DevelopersCorner/build_documentation.md"
+		],
```

Comment:
I don't think having an extra level is useful here, so how about:
```suggestion
		"DevelopersCorner/styleguide.md",
		"DevelopersCorner/build_documentation.md"
```
Also, please do not use tabs (here or anywhere else)